src/schema/core.yaml

```diff
@@ -207,6 +207,19 @@ classes:
           syntax: "{id_nmdc_prefix}:frsite-{id_shoulder}-{id_blade}$"
           interpolated: true
 
+  Sample:
+    class_uri: nmdc:Sample
+    is_a: MaterialEntity
+    description: >-
+    slot_usage:
+      id:
+        required: true
+        description: An NMDC assigned unique identifier for a biosample or processed sample.
+        structured_pattern:
```

Comment:
TODO: figure out why LinkML breaks validation if the parent class specifies a structured_pattern.
TODO: this should be abstract.